deployment/chainloop/templates/controlplane/config.configmap.yaml

```diff
@@ -43,4 +43,8 @@ data:
     {{ if .Values.controlplane.onboarding }}
     onboarding:
       {{- toYaml .Values.controlplane.onboarding | nindent 6 }}
-    {{- end }}
\ No newline at end of file
+    {{- end }}
+    {{ if .Values.controlplane.prometheus_integration }}
+    prometheus_integration:
```

Comment:
prometheus_integration sound very generic, this could mean anything.

What about `prometheus_org_metrics`